profile/README.md

```diff
@@ -4,40 +4,55 @@
 
 ---
 
-## LocalStack ✨
+[![Slack](https://img.shields.io/badge/Slack-@localstack-purple.svg)](https://localstack.cloud/slack)
+[![Twitter](https://img.shields.io/badge/Twitter-@localstack-9cf.svg)](https://twitter.com/localstack)
+[![LinkedIn](https://img.shields.io/badge/LinkedIn-@localstack-darkblue.svg)](https://www.linkedin.com/company/localstack-cloud)
+[![YouTube](https://img.shields.io/badge/YouTube-@localstack-red.svg)](https://www.youtube.com/@localstack)
+[![Discuss](https://img.shields.io/badge/Discuss-@localstack-white.svg)](https://discuss.localstack.cloud)
 
-LocalStack is an easy-to-use test/mocking framework for developing cloud applications. Using LocalStack, you can spin up a local test environment in seconds, and get the same functionality you would get from a real AWS environment.
+## What is LocalStack? 🤔
 
-We are trusted by 10,000s of developers worldwide. Check out our [GitHub repository](https://github.com/localstack/localstack), or search on [Twitter](https://twitter.com/search?q=localstack)/[Google](https://www.google.com/search?q=LocalStack+AWS) to find out what people have to say about LocalStack. It's the modern way of developing cloud applications - we hope you're part of it!
+[LocalStack](https://localstack.cloud) is a cloud service emulator that runs in a single container on your laptop or in your CI environment. With LocalStack, you can run your AWS applications or Lambdas entirely on your local machine without connecting to a remote cloud provider! Whether you are testing complex CDK applications or Terraform configurations, or just beginning to learn about AWS services, LocalStack helps speed up and simplify your testing and development workflow.
```

Comment:
When introducing LocalStack in a general context (like here), I would prefer the notion of "cloud development platform" rather than the more narrow "cloud service emulator" (which is only a sub-aspect of the overall product offering).

Perhaps something like this could work? (feel free to adjust..)
```suggestion
[LocalStack](https://localstack.cloud) is a cloud development platform that facilitates building/testing of cloud and serverless applications on your local machine. At its core, it features a service emulator that runs in a single container on your laptop or in your CI environment. With LocalStack, you can run your AWS applications or Lambdas entirely on your local machine without connecting to a remote cloud provider! Whether you are testing complex CDK applications or Terraform configurations, or just beginning to learn about AWS services, LocalStack helps speed up and simplify your testing and development workflow.
```